src/app/dashboard/dashboard.interfaces.ts

```diff
@@ -0,0 +1,6 @@
+import { IDevice } from '../devices/devices.interfaces';
+
+export interface IWidget {
```

Comment:
In this case will be  better to not use interface. Please have look into my comment for devices.interface